### DISCUSSION TITLE
Shopware Support

### DISCUSSION THREAD

**Discussion by KartoffelToby** [coollabsio/coolify#3377] [Service Template Requests]
A Service with Shopware CE

**Comment by vasani-arpit** [comment]
@andrasbacsai Please add new service label 

project: https://github.com/shopware/platform

**Comment by compgeniuses** [comment]
@KartoffelToby   Kindly Add and Vote for this request here: https://feedback.coolify.io/

**Comment by andrasbacsai** [comment]
Closing this one in favor of: https://feedback.coolify.io/posts/66/service-shopware

**Comment by nyxb** [comment]
is it correct that the link is dead or can only internal people vote?

**Comment by designorant** [comment]
This is a duplicate of: https://github.com/coollabsio/coolify/discussions/3304

**Comment by Cinzya** [comment]
Just added a example for Shopware 6 to our [official Coolify examples](https://github.com/coollabsio/coolify-examples/tree/v4.x/shopware6).

This was based on [Shopwares own example repository](https://github.com/shopwareLabs/example-docker-repository), with slight changes to support Coolify envs and a few missing dependencies for Shopware 6.7. This wasn't tested yet against every scenario, so might need some further changes, but from my initial test, it builds and runs fine to me in a [v4.0.0-beta.453](https://github.com/coollabsio/coolify/releases/tag/v4.0.0-beta.453) coolify instance.

Let us know if you have any suggestions on how we can improve it or if you face any errors.
